boards/xtensa/esp32/common/src/esp32_board_spi.c

```diff
@@ -71,7 +71,7 @@ static inline uint8_t spi_status(struct spi_dev_s *dev, uint32_t devid)
 static inline int spi_cmddata(struct spi_dev_s *dev, uint32_t devid,
                               bool cmd)
 {
-#ifdef CONFIG_LCD_ILI9341
+#if defined(CONFIG_LCD_ILI9341) || defined(CONFIG_LCD_SSD1680)
```

Comment:
Let's change line 55 in this file as well.

Reply:
I Would like to use **CONFIG_SSD1680_GPIO_PIN_DTA_CMD** instead of **DISPLAY_DC**. The **CONFIG_SSD1680_GPIO_PIN_DTA_CMD** is provided by Kconfig. Do I need to use DISPLAY_DC ?

Reply:
What is the motivation for having it as a Kconfig option?
I explained why I consider it as a bad option [in another comment](https://github.com/apache/incubator-nuttx/pull/6511#discussion_r906054724), which has not been addressed.

Reply:
Having that fixed per board is the best I think. I agree with @gustavonihei

Reply:
OK. Now I use only DISPLAY_DC, there is no more CONFIG_SSD1680_GPIO_PIN_DTA_CMD.